wave_vision/include/wave/vision/tracker/tracker.hpp

```diff
@@ -0,0 +1,171 @@
+/**
+ * @file
+ * Brute force matcher implementation, derived from descriptor matcher base.
+ * @ingroup vision
+ */
+#ifndef WAVE_VISION_TRACKER_HPP
+#define WAVE_VISION_TRACKER_HPP
+
+#include <chrono>
+#include <string>
+#include <vector>
+
+#include <opencv2/opencv.hpp>
+
+#include "wave/containers/landmark_measurement.hpp"
+#include "wave/containers/landmark_measurement_container.hpp"
+#include "wave/utils/utils.hpp"
+#include "wave/vision/utils.hpp"
+
+namespace wave {
+/** @addtogroup vision
+ *  @{ */
+
+class FeatureTrack {
+ public:
+    // Constructor
+    FeatureTrack() {}
+
+    // Initializer list constructor
```

Comment:
Sorry, will remove. I like to think I'm not writing redundant comments anymore but apparently a few still make it through :stuck_out_tongue: